src/v2/view-builder/views/device/DeviceChallengePollView.js

```diff
@@ -145,6 +147,24 @@ const Body = BaseFormWithPolling.extend(
       });
     },
 
+    showCustomFormErrorCallout(error) {
+      const errorSummary = error.responseJSON.errorSummary;
+      let options = {
+        type: 'error',
+        className: 'okta-verify-uv-callout-content',
+        subtitle: errorSummary,
+      };
+
+      const containsSignedNonceError = error.responseJSON.errorSummaryKeys
```

Comment:
Is this error response is not standard IDX response ?

Reply:
updated